{
  "0 debug pnpm:scope": {
    "selected": 1
  },
  "1 info pnpm": {
    "err": {
      "name": "Error",
      "message": "Command failed with ENOENT: intsall\nspawn intsall ENOENT",
      "code": "ENOENT",
      "stack": "Error: Command failed with ENOENT: intsall\nspawn intsall ENOENT\n    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)\n    at onErrorNT (internal/child_process.js:469:16)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"
    }
  },
  "2 error pnpm": {
    "errno": "ENOENT",
    "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
    "syscall": "spawn intsall",
    "path": "intsall",
    "spawnargs": [],
    "originalMessage": "spawn intsall ENOENT",
    "shortMessage": "Command failed with ENOENT: intsall\nspawn intsall ENOENT",
    "command": "intsall",
    "escapedCommand": "intsall",
    "failed": true,
    "timedOut": false,
    "isCanceled": false,
    "killed": false,
    "prefix": "/home/workspace/reactjs/coding-practices/appointmentsApp",
    "err": {
      "name": "Error",
      "message": "Command failed with ENOENT: intsall\nspawn intsall ENOENT",
      "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
      "stack": "Error: Command failed with ENOENT: intsall\nspawn intsall ENOENT\n    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)\n    at onErrorNT (internal/child_process.js:469:16)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"
    }
  }
}